components/mpas-framework/Makefile

```diff
@@ -1159,11 +1166,11 @@ errmsg:
 	@echo "    SHAREDLIB=true - generate position-independent code suitable for use in a shared library. Default is false."
 	@echo "    USE_LAPACK=true - builds and links with LAPACK / BLAS libraries.  Default is to not use LAPACK."
 	@echo "    USE_SHTNS=true - builds and links with SHTNS/FFTW.  Default is false."
+	@echo "    HAVE_ADIOS=false   - PIO2 library does not include ADIOS support.  Default is true."
```

Comment:
You would need to change this language.

Reply:
I believe this is really what we want.  The default is that you have ADIOS and you have to specify `HAVE_ADIOS=false` when building to indicate that your PIO version doesn' support it.

Reply:
Yes you're right.